serve/mlc_serve/run.py

```diff
@@ -79,38 +75,41 @@ def setup_logging(args):
 def create_engine(
     args: argparse.Namespace,
 ):
+    # `model_artifact_path` has the following structure
```

Comment:
We should probably put this in a doc string somewhere, can also be a follow up item.